src/components/CourseCard/CourseCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { CourseCardProps } from "../../type/CourseCardSituation";
+import styles from '../CourseCard/CourseCard.module.css';
+const CourseCard: React.FC<CourseCardProps & { className: string }> = ({ image, profession, price, description, rating, numPeople, className }) => {
+
+    const renderStars = (rating: number) => {
+        const stars = [];
+        for (let i = 0; i < 5; i++) {
+            stars.push(
+                <img
+                    key={i}
+                    src={i < rating ? "star-filled.png" : ""}
+                    alt={i < rating ? "Filled Star" : ""}
+                    className={styles['rating-star']} />
+            );
```

Comment:
Bu kısımda for 2 rating olsa bile her item için 5 kere dönecek. Bunun yerine for'u rating numarası kadar döndürebilirsin.